Source/NSPopover.m

```diff
@@ -34,13 +34,115 @@
 #import "AppKit/NSPopover.h"
 #import "AppKit/NSViewController.h"
 #import "AppKit/NSView.h"
-#import "AppKit/NSWindow.h"
+#import "AppKit/NSPanel.h"
+#import "AppKit/NSNibLoading.h"
+#import "AppKit/NSStoryboard.h"
+#import "AppKit/NSGraphics.h"
 
+// Popover private classes
+
+@interface GSPopoverView : NSView
+
+- (void) setContentView: (NSView *)view;
+
+@end
+
+@implementation GSPopoverView
+
+- (instancetype) initWithFrame: (NSRect)frame
+{
+  self = [super initWithFrame: frame];
+  if (self)
+    {
+      [self setAutoresizingMask: NSViewWidthSizable | NSViewHeightSizable];
+    }
+  return self;
+}
+
+- (void) setContentView: (NSView *)view
+{
+  if (view != nil)
```

Comment:
I still don't think this method ever gets Calle. Could you please add an NSLog statement here and see if it gets printed out?

Reply:
You are correct.  I removed the method after testing it, as you suggested.